components/aboutcontent.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const About = props => {
```

Comment:
The filename is not the same as the component name. Also filenames for components should be PascalCase.